mise.toml

```diff
@@ -3,3 +3,4 @@ dotnet = "9"
 java = "24"
 node = "24"
 rust = "1.90.0"
```

Comment:
note: this was already here, and previously we were using rust 1.70 in CI, but figured it's probs fine to bump to v1.90